.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "automatic"
+}
```

Comment:
This file is removed from https://github.com/awslabs/amazon-sns-java-extended-client-lib/pull/17/commits/541d253746a36622e0945c149fae7810950aeaf8 and added back here. Can you combine all commits together?

Reply:
Yes will squash all commits together.